### DISCUSSION TITLE
Dropdown menu for pending devices in the device list page

### DISCUSSION THREAD

**Discussion by benasse** [shellhub-io/shellhub#2559] [Ideas]
In the first version of the interface, in the drop-down menu of the pending devices, it was possible to consult the information of the device.

This is no longer possible in v2.

I quickly looked at it does not seem to have any technical limitation to display the information.

What do you think about putting the information back in the v2?

**Comment by gustavosbarreto** [comment]
@benasse,

It seems like a regression and that the change in behavior was not intentional. Thanks for pointing it out.

I have investigated the issue and agree that there doesn't seem to be any technical limitation preventing us from displaying the information. I will create an issue to prioritize adding the device information back into the v2 interface as soon as possible.

Thank you again for your feedback.

**Comment by otavio** [comment]
Issue to track this issue is #2560
